.devcontainer/dev/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "uyuni-dev-container",
+  "build": { "dockerfile": "Dockerfile" },
+  "remoteEnv": {
+    "JAVA_HOME": "/usr/lib64/jvm/java-17-openjdk"
+  }
+  // More info: https://containers.dev/implementors/json_reference/
```

Comment:
in my test I added 
```
  "postCreateCommand": "cd /workspaces/uyuni &&  ant -f java/manager-build.xml ivy"
```
in this way when the container is created, the dependencies are automatically downloaded